db.js

```diff
@@ -1,4 +1,13 @@
-{
+/**
+ * @module codes-db
+ */
+
+/**
+ * Database of all HTTP Codes
```

Comment:
What is an HTTPCode type?

How do you document the structure?

It is of type object